app/views/shared/check_answers/_benefits_breakdown.html.erb

```diff
@@ -4,7 +4,7 @@
 <%= govuk_summary_list(actions: false) do |summary_list| %>
   <% @legal_aid_application.applicant.state_benefits.each do |benefit| %>
     <% summary_list.with_row do |row|
-         row.with_key { benefit.description }
+         row.with_key { benefit.description || "" }
```

Comment:
would safe operator work, but either way 
```suggestion
         row.with_key { benefit&.description }
```

Reply:
The summary list helpers break if you try and pass `nil` that's why I ended up on the empty string :+1: